<launch>
	<arg name="robot_id"/>
	<arg name="robot_tf"/>
	<arg name="init_pose"/>

    <!-- load params -->
    <rosparam command="load" file="$(find ar_commander)/configs/sim_params.yaml"/>
    <rosparam command="load" file="$(find ar_sim)/configs/sim_interface_params.yaml"/>
    <rosparam command="load" file="$(find ar_commander)/configs/robot_v1.yaml"/>

	<!-- Set paths to repos -->
    <param name="AR_COMMANDER_DIR" value="$(find ar_commander)" />
	<param name="AR_SIM_DIR" value="$(find ar_sim)" />

	<!-- Spawn robot in sim -->
	<include file="$(find ar_sim)/launch/gazebo_sim.launch" >
		<arg name="init_pose" value="$(arg init_pose)" />
		<arg name="robot_id" value="$(arg robot_id)" />
	</include>

	<!-- Launch autonomy stack -->
	<include file="$(find ar_commander)/launch/autonomy_stack.launch" />

</launch>
